go-app.js

```diff
@@ -29,12 +30,21 @@ go.app = function() {
             });
         });
 
-        self.states.add('states:end', function(name) {
+        self.states.add('states:tts', function(name) {
             return new EndState(name, {
-                text: 'Thanks, cheers!',
+                text: 'Oak is strong and also gives shade.',
                 next: 'states:start'
             });
         });
+
+        self.states.add('states:recorded', function(name) {
+            // TODO: Add voice URL to message helper_metadata
+            return new EndState(name, {
+                text: '',
```

Comment:
Perhaps we should fill in some text here like ``If you are hearing this, playing the recording has failed.``?